x11-apps/fusion-icon/fusion-icon-0.2.4-r1.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 eutils gnome2-utils
+
+DESCRIPTION="Fusion Icon (Compiz tray icon) for Compiz 0.8.x series"
+HOMEPAGE="https://github.com/compiz-reloaded"
+SRC_URI="https://github.com/compiz-reloaded/${PN}/releases/download/v${PV}/${P}.tar.xz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk2 gtk3 qt4 qt5"
+REQUIRED_USE="?? ( gtk2 gtk3 ) ?? ( qt4 qt5 )  || ( gtk2 gtk3 qt4 qt5 )"
+
+RDEPEND="
+	>=dev-python/compizconfig-python-0.8.12[${PYTHON_USEDEP}]
+	<dev-python/compizconfig-python-0.9
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	x11-apps/xvinfo
+	x11-apps/mesa-progs
+	>=x11-wm/compiz-0.8
+	<x11-wm/compiz-0.9
+	gtk2? (
+		dev-libs/libappindicator:2
+		x11-wm/compiz[-gtk3]
+	)
+	gtk3? (
+		dev-libs/libappindicator:3
+		x11-wm/compiz[gtk3]
+	)
+	!gtk3? ( x11-wm/compiz[gtk(+),-gtk3] )
```

Comment:
Is this actually needed? As far as I know compiz uses gtk only for decorations.

Reply:
One can install `fusion-icon` with `gtk2` alongside `compiz` with `gtk3` or vice versa. They are using gtk independently from each other so there shouldn't be any issues. You can drop lines `29`, `33` and `35`.